.github/workflows/tests.yml

```
@@ -34,10 +34,15 @@ jobs:
           pip install -U pip wheel
           pip install -U -c constraints.txt -r requirements-dev.txt qiskit-terra numpy scipy
         shell: bash
+      - name: Run clang-format
+        run: |
+          set -e
+          sh tools/clang-format.sh --Werror -n
+        shell: bash          
       - name: Run Lint
         run: |
           set -e
-          pycodestyle --ignore=E402,W503,W504 --max-line-length=100 qiskit_aer
+          black --check qiskit_aer test tools
```

This is missing the `setup.py` you have in the tox config and it will fail the `tox -elint` job locally because it needs to be reformatted.

```suggestion
          black --check qiskit_aer test tools setup.py
```